contrib/platforms/macosx

```diff
@@ -18,11 +18,9 @@ if [ "x$enable_fortran" != xno -a ! -x "$(command -v "$CC")" ]; then
   done
 fi
 
-# Per-user personalizations
-if [ "x$USER" == "xbosilca" ]; then
-    with_hwloc=${HWLOC_ROOT:=/Users/bosilca/opt}
-    #with_cuda=${CUDA_ROOT:=/Developer/NVIDIA/CUDA-9.1/}
-    with_ayudame=${AYUDAME_ROOT:="/Users/bosilca/opt/temanejo"}
+# OS-X 12.2 provides Bison 2.3, we need Bison 3 or better
```

Comment:
let's not overspecialize (macport installs in /opt/local/). But please remove the personalization above.

Reply:
Ok, I took into account the other common packagers, MacPorts and Fink. I have Brew here, can you try that MacPort does what it should?